### DISCUSSION TITLE
New man on the blok :)

### DISCUSSION THREAD

**Discussion by Plovmann** [martin2250/OpenCNCPilot#173] [General]
Hallo All.  
Im a new user of OpencncPilot, and I find it great.  I dont even know  about such program :)   Im normerly just du in wood on my CNC. 

My name is Lars, Im from Denmark

**Comment by martin2250** [comment]
Hi Lars, thanks for introducing yourself. Good to have you here!

I'm moving this to discussions, so it can stay open.

Cheers!

**Comment by Wiki-the-Tinkerer** [comment]
Hi, I'm also new to the openCNCPilot - but i have to say it is a brilliant piece of work! So a big thank you to Martin, and all involved. This is so much better than any of the usual GRBL suspects available. I used to be a UI gui, so will dump a couple of questions in a new thread. 
/Wiki
